lib/puppet/face/epp.rb

```diff
@@ -291,8 +303,8 @@
       summary "A .pp or .yaml file that is processed to produce a hash of values for the template."
     end
 
-    option("--facts <yaml_file>") do
-      summary "A .yaml file containing a hash of facts made available in $facts"
+    option("--facts <yaml_file>[<json_file>]") do
```

Comment:
The option argument description looks a bit off. Shouldn't it be `"--facts [<yaml_file>|<json_file>]"` ? Or does the current syntax have specific meaning for the options parser?